wercker.yml

```diff
@@ -51,7 +51,7 @@ build:
           # TODO: Remove exclusions on gas checks
           # G101: Hardcoded credentials (Presently only false positives)
           # G204: Unsafe command execution (Presently only affects build tools)
-          gas -quiet -exclude=G101,G204 ./{cmd,core}/**/*.go
+          gas -quiet -exclude=G101,G204 -skip="*vendor*" -skip="*syscall*" -skip="*_test.go" ./...
```

Comment:
(Not sure if my comment on the other thread is effectively buried, so repeating here.)

Actually ideally it would be just [zsyscall_windows.go]. (Or I guess files in syscall that start with 'z' which appears to be the convention for the tool that generates this stuff.)